### DISCUSSION TITLE
How do I stop a fraudulent Safari warning on a legitimate website?

### DISCUSSION THREAD

**Discussion by meowmewan** [community/community#135671] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have a site that Safari has marked as fraudulent. Only Safari! All other browsers perceive it normally. There is an SSL certificate, security is configured, according to all checkers and screeners the site is completely clean and safe.

The Apple "Report a bug" form has been filled out a million times already. I have talked to more than 100 people and no one has ever encountered this. The only case that exists on the Internet is that the t.me domain was blocked in the same way, but it was unblocked in 2 hours, and my site has been blocked since August 8.

Has anyone encountered this?

<img width="1680" alt="Снимок экрана 2024-08-08 в 10 35 23" src="https://github.com/user-attachments/assets/cb5725c7-2faa-4dec-97a6-8c0edaca59b4">

**Comment by erencanbas** [comment]
**Safari's security features** are often tied to Apple’s proprietary systems. Ensure your site isn’t mistakenly flagged by any of the third-party services Apple relies on, like Google's Safe Browsing API.

**Reputation Services:** Apple may also use reputation services that aggregate information from various sources. If your site recently changed its IP address, domain name, or hosting provider, this could have caused a temporary flag in their system.

**ISP or DNS Propagation:** Sometimes, local DNS servers or specific ISPs might have cached incorrect or outdated security data about your site. Flushing the DNS cache might help, though this is more a user-side fix.
**Network Level Blocking**: It's also worth checking if your site is inadvertently blocked or flagged at a network level by major ISPs, as these could influence Safari’s detection.

**Reply by meowmewan** [reply]
All of the above works properly, there are no problems with it.

**Comment by Nihal-A-Khan** [comment]
Hello @meowmewan 

**Here are some of the things you can consider doing if you have not done so already:**

1. Check SSL Configuration: Ensure your SSL certificate is correctly installed and configured. Tools like SSL Labs’ SSL Test can help verify this.
Domain Reputation: Verify your domain’s reputation across multiple security services. Use services like VirusTotal, Web of Trust (WOT), or Google Safe Browsing to see if any reports are flagging your domain.

2. Apple’s Safari Support: Since you mentioned filling out the “Report a bug” form, try contacting Apple Support directly. Sometimes, reaching out to their support team through other channels like their [Apple Developer Support](https://developer.apple.com/contact/) or [Apple Support Communities](https://discussions.apple.com/welcome) might expedite the process.

3. Review Logs: Check your site’s logs for any unusual activity or signs of abuse that might have triggered a false positive.
Security Plugins: Ensure no security plugins or services are misconfigured in a way that could trigger false flags.

4. Google Search Console: Ensure your site is verified with Google Search Console and check if any issues are reported there.
Bing Webmaster Tools: Similarly, verify your site with Bing Webmaster Tools to check for any security or reputation issues.

**Note:** This process involves certain risks and may not guarantee the desired outcome. Proceed with your own discretion and caution. I cannot be held responsible for any issues or complications that arise from following this method. You assume all responsibility for the results.

If you have any further questions or if this process works for you, please let me know! If it resolves your issue, please mark this response as the answer! Thank you in advance!

**Reply by meowmewan** [reply]
Thanks for the answer!

1. SSL is configured correctly and the test showed it
2. Domain reputation in VirusTotal and Google Safe Browser is in the green zone, everything is clean
3. I live in Russia, in my region it is impossible to directly contact Apple support. I wrote to them in the online chat, they can’t help me there. I also wrote to the Apple community, they can’t help me either
4. There is nothing unusual according to the site logs
5. Security works normally
6. The site has been checked in Google Search Console, Bing Webmaster Tools, Yandex Webmaster - there are no problems or errors

The most important and offensive thing is that I cannot directly contact Apple support and, if possible, leave a request to remove my site from the block list

**Comment by antonxklimov** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> I have a site that Safari has marked as fraudulent. Only Safari! All other browsers perceive it normally. There is an SSL certificate, security is configured, according to all checkers and screeners the site is completely clean and safe.
> 
> The Apple "Report a bug" form has been filled out a million times already. I have talked to more than 100 people and no one has ever encountered this. The only case that exists on the Internet is that the t.me domain was blocked in the same way, but it was unblocked in 2 hours, and my site has been blocked since August 8.
> 
> Has anyone encountered this?
> 
> <img alt="Снимок экрана 2024-08-08 в 10 35 23" width="1680" src="https://private-user-images.githubusercontent.com/178306396/357464496-cb5725c7-2faa-4dec-97a6-8c0edaca59b4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTMyODg4NDcsIm5iZiI6MTc1MzI4ODU0NywicGF0aCI6Ii8xNzgzMDYzOTYvMzU3NDY0NDk2LWNiNTcyNWM3LTJmYWEtNGRlYy05N2E2LThjMGVkYWNhNTliNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyM1QxNjM1NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YWFiN2M2YzY5Mzg2NzFjZDcwYjdiYTc3ODc1MjBjN2M1NjM3OGYxYWZkNmVhYTkyMzhkNmJjMGM1OGQ2ZGNmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LPa72Ps0-WjMK6mHHy9JnctQ93g8ru_IruUQ7_ixV1c">

Did you happen to resolve this issue?
